tasks.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
If we didn't require python 3, we do now.
That needs to be listed on the main, and the version tested with / supported needs to be marked.
Also, I'm not sure pipenv can be assumed, please clarify

Reply:
Only pipenv and python3 are required, and it's only to do releases, not for installing or running.

Reply:
Maintainers.md then